.github/workflows/flowdag_schema_check.yml

```diff
@@ -22,7 +22,6 @@ jobs:
         shell: pwsh
         env:
           PYTHONPATH: ${{ github.workspace }}/src/promptflow
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
so we don't use this inside our script?

Reply:
this is used in previous version of publish. it is not used now.